### ISSUE TITLE
Add support to remap buttons

### ISSUE THREAD

**Issue by jchaike** [juharris/switch-remoteplay#58]
To test out `switch-remoteplay`, I plugged in a off-brand wired USB controller.
It works for the most part to test, but I noticed some of the button mappings were off.

It would be great if the web client supported a remapping function to ensure as many controllers can work as possible.

Thanks!

**Comment by juharris** [comment]
Yeah this would be great. I'm not really using this tool anymore but if someone wants to submit a pull request for this and I'm happy to review it.

**Comment by juharris** [comment]
Are you using this in a desktop browser? Maybe there's something quick that I can do.
